File: 2044.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var n = 0;
while(n != -1){
    n = parseInt(lines.shift());
    if(n > -1){
        let entries = lines.shift().split(' ');
        let sum = 0, count = 0;
        for(let i=0; i<entries.length; i++){
            let entry = parseInt(entries[i]);
            sum += entry;
            if(sum % 100 == 0){
                count++;
            }
        }
        console.log(count)
    }
}

File: 2044.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){

    int n;
    while(cin >> n && n > 0){
        int val, sum = 0, count = 0;
        while(n--){
            cin >> val;
            sum += val;
            if(sum % 100 == 0) count++;
        }
        cout << count << endl;
    }

    return 0;
}